src/core/agent/agent.ts

```diff
@@ -91,10 +91,24 @@ class AriesAgent {
   async start(): Promise<void> {
     if (!AriesAgent.ready) {
       await this.basicRecordStorage.open(config.walletConfig?.id || "idw");
-      await this.signifyApi.start();
-      AriesAgent.ready = true;
+      try {
+        await this.signifyApi.start();
+        AriesAgent.ready = true;
+      } catch (error) {
+        AriesAgent.ready = false;
+        throw error;
+      }
```

Comment:
I don't think there's a need for this try catch. `AriesAgent.ready` will be `false` if something throws, since it's only set to true at the end.